py/server/deephaven/updateby_clause.py

```diff
@@ -0,0 +1,269 @@
+#
```

Comment:
Overall, I have the feeling that the updateby nomenclature is not as well hashed out as something like `agg`.  For example:
1) the `updateby_clause` module name feels much worse than `agg`.  would we be better off with `updateby`, `update_by`, or something else?
2) Nomencluature is inconsistent with other modules.  e.g. `pairs` seems to be roughly the same thing as `cols` in `agg`.
3) Is "clause" the right word?  In the docs, they are called "operators".  What would convey the concept best to users.  Same comment applies to the `update_by` argument name.